modules/ROOT/pages/upgrade-service-mesh.adoc

```diff
@@ -0,0 +1,60 @@
+= Upgrade Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, upgrading
+
+
+Anypoint Service Mesh v1.1 includes new features and architecture enhancements, such as bypassing the Istio Mixer when sending requests from clients to Anypoint Platform. This change caused an Anypoint Service Mesh architecture updates, and therefore, introduces a downtime for the Anypoint Service Mesh upgrade to v1.1.
+
+Anypoint Service Mesh v1.1 includes new features and architecture enhancements, such as stopping using Istio Mixer. Because of this change, upgrading from Anypoint Service Mesh v1.0 to v1.1 introduces downtime during the c
+
+== Prerequisites
+
+Before you upgrade, ensure that you meet the following requirements:
+
+. Ensure that the current Istio version in your environment is at least 1.6.0 (control plane and data plane):
+[source,text,linenum]
+----
+$ istioctl version
+client version: 1.6.0
+control plane version: 1.6.0
+data plane version: 1.6.0 (5 proxies)
+----
+. Download the latest Anypoint Service Mesh command line interface (CLI) version:
++
+`curl -Ls http://anypoint.mulesoft.com/servicemesh/xapi/v1/install > asmctl && chmod +x asmctl`
```

Comment:
This is a task that is (should be) documented elsewhere, as a step.

Reply:
Done! Thanks!